frontend/src/components/forms/supplier/Supplier.tsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react'
 
 //component imports
+import CustomSelect from "../_elements/custom-select/CustomSelect";
 
+import Contact from "../_elements/contact/Contact";
 //interface imports
+import {IFormProps} from "../../../interfaces/INewItem";
+import {Weekdays} from "../../../interfaces/weekdays";
 
-type Props = {};
+type Props = IFormProps;
 
-function Supplier(props: Props){
+function Supplier({handleChange}: Props){
+    const props = {handleChange: handleChange, model: "supplier"}
     return(
-       <div>Supplier</div>
+        <div>
+            <Contact {...props}/>
+            <CustomSelect name="weekdays" label={"Weekdays"} options={Object.values(Weekdays)} {...props}  />
```

Comment:
Does it make sense to combine curly braces and quotes as in the label here? Either it's a variable or a string, isn't it?

Reply:
the curly braces just indicate that javascript is being used here. strings are strings in javascript so it works even though it's redundant. They get autogenerated by intellij while typing and sometimes I forget/am to lazy to remove them